lib/offsite_payments/integrations/realex_offsite.rb

```diff
@@ -115,11 +234,13 @@ def initialize(order, account, options = {})
           add_field 'AUTO_SETTLE_FLAG', '1'
           add_field 'RETURN_TSS', '1'
           add_field 'TIMESTAMP', @timestamp
+          add_field 'HPP_VERSION', '2'
           # Realex does not send back CURRENCY param in response
           # however it does echo any other param so we send it twice.
           add_field 'X-CURRENCY', @currency
           add_field 'X-TEST', @test.to_s
           add_field 'ORDER_ID', "#{order}#{@timestamp.to_i}"
+          add_field 'COMMENT1', 'Shopify'
```

Comment:
Could you use `application_id` here? This library is open source and could be used by others that are not Shopify. We will set `OffsitePayments::Integrations::RealexOffsite::Helper.application_id` to `Shopify` on our side.

Reply:
@florinionce Could you help here?